server/src/services/app_import_export.service.ts

```diff
@@ -385,6 +385,7 @@ export class AppImportExportService {
           kind: source.kind,
           type: source.type || DataSourceTypes.DEFAULT,
           appVersionId: appVersionMapping[appVersion.id],
+          pluginId: source?.pluginId || null,
```

Comment:
will it work for inter-instance imports?
the `source?.pluginId` of an instance won't be available with other